app/workers/issue_reminder_worker.rb

```diff
@@ -0,0 +1,65 @@
+module Pushbit
+  class IssueReminderWorker < BaseWorker
+
+    def work(user_id)
+      user = User.find(user_id)
+      issues = {}
+
+      b = Pushbit::Behavior.find_by!(kind:"issue_reminder")
+
+      b.repos.joins(:users).where(["users.id = ?", user.id]).all.each do |r|
+        client.list_issues(r.github_full_name).each do |i|
+
+          t = i.pull_request.nil? ? "issue" : "PR"
+
+          cs = client.issue_comments(r.github_full_name, i['number'])
+          cs = cs.reject { i.user[:login] == "pushbit-bot" }
+
+          last_updated_at = i.created_at
+
+          if cs.length > 0 
+            last_updated_at = cs.last.created_at
+          end
+
+          if i['created_at'] < 60.days.ago
+            client.add_comment(
+              r.github_full_name, 
+              i['number'], 
+              "We've closed this #{t} due to inactivity"
```

Comment:
Can we move these comments to separate template files? I'd like to make the messages a bit longer and friendlier now that the base work is in place 😄